resources/lang/de_DE/default.po

```diff
@@ -1226,8 +1226,14 @@ msgstr "iCal Export"
 msgid "JSON Export"
 msgstr "JSON Export"
 
-msgid "Your night shifts between %d and %d am count twice for the %s score."
-msgstr "Deine Nachtschichten zwischen %d und %d Uhr zählen für den %s Score doppelt."
+msgid "Your night shifts between %d and %d am are multiplied by %d for the %s score."
+msgstr "Deine Nachtschichten zwischen %d und %d Uhr werden für den %4$s Score mit %3$d multipliziert."
```

Comment:
Eventually we could just use the generic sentence and thus remove the `Your ` here?

Reply:
done

Reply:
*is missing a pfusch*